airflow/contrib/operators/sqoop_operator.py

```diff
@@ -168,16 +194,18 @@ def execute(self, context):
                     split_by=self.split_by,
                     where=self.where,
                     direct=self.direct,
-                    driver=self.driver)
-            elif not self.query:
+                    driver=self.driver,
+                    extra_import_options=self.extra_import_options)
+            elif self.query:
```

Comment:
was `elif not self.query` but should be `elif self.query` because when a query is passed it should execute the `import_query` logic

Reply:
I checked the commit history, and you're right.